layouts/default.vue

```diff
@@ -546,7 +559,14 @@ export default {
               :show-header="!g.isRoot"
               @selected="groupSelected($event)"
               @expand="groupSelected($event)"
-            />
+            >
+              <template #header>
+                <h6 class="root-group">
```

Comment:
Are we using `h6` to size content or does this truly represent a section heading?